### DISCUSSION TITLE
Incorrect charge count

### DISCUSSION THREAD

**Discussion by DFOUD** [teslamate-org/teslamate#3572] [Q&A]
There seems to be an ungodly number of charges on the update page, how do I fix it, I need help!
![image](https://github.com/teslamate-org/teslamate/assets/44726767/6dba53e6-9019-4314-a247-04f8f2192b4e)

**Comment by cwanja** [comment]
How long were you on that software update for (column to the left)? You can check against the Charges dashboard in the same timeline. But the count is coming from the same data source as the Charges, so I imagine that the data is "correct".

**Comment by DFOUD** [comment]
Unlikely, can't really do 471 charges in a couple of weeks, my charging habit is simply to fill it up every day.
![IMG_6897](https://github.com/teslamate-org/teslamate/assets/44726767/0e9662ff-968f-4a51-80b8-d92b6377a00b)

**Reply by cwanja** [reply]
What does your Charges dashboard show during that timeframe?

**Reply by DFOUD** [reply]
I believe 10 charge counts is a reasonable figure
![IMG_6900](https://github.com/teslamate-org/teslamate/assets/44726767/584f5e4f-dac4-4fd3-9f6a-863deff61e28)
